src/core/stream_send.c

```diff
@@ -1365,6 +1372,26 @@ QuicStreamOnLoss(
 
     if (AddSendFlags != 0) {
 
+        // Check stream's 'cancel on loss' flag to determine how to handle the resends queued up at this point.
```

Comment:
```suggestion
        //
        // Check stream's 'cancel on loss' flag to determine how to handle the resends queued up at this point.
        //
```
nit: style. Also, please wrap at 80 characters.